### PR TITLE
Aboutsection

### PR DESCRIPTION
**Pull request by aainagargg**
Added more details, changed the about section from the README to information about our tool. @YesIfwRONG to make some minor changes before this branch can be merged.

### CODE REVIEW

File: view/src/components/BiasDifferenceDisplay.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ export default function BiasDifferenceDisplay({
   dataset2,
 }: BiasDifferenceDisplayProps): JSX.Element {
   return (
+
```
Comment (CodeYimin) [inline view/src/components/BiasDifferenceDisplay.tsx:15]: No need for space here

File: view/src/components/TableOfContents.tsx [hunk 2]
```diff
@@ -2,31 +2,35 @@ import { ReactElement } from "react";
...
-        </li>
-        <li>
-          <a href="#project-structure">Project Structure</a>
-        </li>
-        <li>
-          <a href="#running-the-project">Running the Project</a>
-        </li>
-      </ul>
-    </>
+      <>
```
Comment (CodeYimin) [inline view/src/components/TableOfContents.tsx:5]: This should be two space indent not four

File: view/package-lock.json
Comment (CodeYimin) [inline view/package-lock.json:1]: Don't change or push the package lock. This is failing the netlify build

**Review by CodeYimin — CHANGES_REQUESTED** [review]

**Comment by CodeYimin** [comment]
Reminder to not push package lock unless you're updating the packages, it is causing netlify build to fail

**Comment by SaahilKapasi** [comment]
A new PR has been made for these changes.

**Pull request closed without merging**